README.md

```diff
@@ -712,6 +713,19 @@ Adjust the volume.
 
 Platforms: all
 
+#### drm
+To configure DRM it is necessary to add the poperties:
+
+`drmName` Which is the DRM type. It can take a value of these DRM's:
+* **widevine**
```

Comment:
How about creating a constant for this:
```
DRMName = {
  CENC: "cenc",
  PLAYREADY: "playready",
  WIDEVINE: "widevine",
};
```

Similar to what we have for FilterType